packages/tools/playground/src/tools/utilities.ts

```diff
@@ -1,6 +1,9 @@
 import type { GlobalState } from "../globalState";
 
 export class Utilities {
+    /** Error message to display when the playground might lose the current code */
+    public static CodeLostConfirmationMessage = "Are you sure you want to switch the language (You will lose your current project if it was not saved before)?";

```

Comment:
I think the message should be adapted when switching engines/versions?

Reply:
Thanks a lot, good catch :-)